Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
I can explain. The bottles we have for formulae are built against stable versions of their dependencies, right? `Formula#installed?` is used to see if a dependency is satisfied. The method can return true even if dependency is not stable potentially breaking the bottle we're pouring. But let's skip it for now. Let's skip it for now and consider #668. To allow this we need to mind dependencies specs that are installed and upgrade them properly. Doing so can break the bottle we pour. So, here is how we can solve the problems:
* We used to resolve dependencies as stable and install them as stable unless they were `satisfied?`. Which caused the #668.
* Instead we decided to mind the specs of installed dependencies and upgrade them properly
* If we pour a bottle and its dependencies are not stable, we should warn and ask user to specify some option:
1. build formula against dependency
2. pour bottle regardless it being broken
3. don't mind installed specs of dependencies and install stable dependencies against which the bottle we provide is built.